src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -187,6 +187,10 @@
                         RelativePanel.Below="AboutImage"
                         Orientation="Vertical" >
 
+                <Button Click="Button_Click" Margin="0,4,0,4">
+                    <TextBlock Text="OOBE"/>
```

Comment:
We should use an other name. I am not think that everyone knows what an OOBE is.
I suggest "First run experience".

 This label should be moved to resx for localisation.

Reply:
@htcfreek 
this is a working branch with temporary stuff that is not what you'll see in the final release. For now that is just a place holder. There will be multiple people working on this branch to fill the gaps.

Reply:
Ooops. Didn't recognized.